monkey/monkey_island/cc/ui/src/components/report-components/security/PostBreach.js

```diff
@@ -62,19 +63,26 @@ class PostBreachComponent extends React.Component {
     });
     let defaultPageSize = pbaMachines.length > pageSize ? pageSize : pbaMachines.length;
     let showPagination = pbaMachines > pageSize;
+    const howManyPBAs = pbaMachines.reduce((accumulated, pbaMachine) => accumulated+pbaMachine["pba_results"].length,0)
```

Comment:
no need to add `,0` at the back.

Reply:
Considered best practice to avoid unexpected `+` behaviour, like concating strings or stuff